ocs_ci/helpers/odf_cli.py

```
@@ -166,6 +167,24 @@ def _set_executable_permissions(self):
         os.chmod(self.local_cli_path, current_permissions | S_IEXEC)
         log.info(f"Set executable permissions for {self.local_cli_path}")
 
+    def _create_fdf_alias(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_create_fdf_alias`

Add return type annotation: `None`

(ANN202)